### DISCUSSION TITLE
Mastercontainer does not create Apache, Nextcloud, Imaginary... containers

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#2759] [Questions] [answered]
Greetings,

I am experiencing a problem with the installation of AIO v6.1.1 on Ubuntu 20.04.6 using the docker-compose file (UFW firewall disabled). I previously followed the same tutorial for another installation a few months ago, which is still functioning correctly.

Allow me to explain the issue I am facing. Upon starting the master-container, it creates new containers; however, some dockers such as Apache, Nextcloud, Imaginary, and Talk recorder are missing.

I have attached logs and screenshots for your review.

What steps should I take to resolve this matter?

#### Other valuable info  attached
[aio-issue.zip](https://github.com/nextcloud/all-in-one/files/11744457/aio-issue.zip)

**Comment by vawaver** [comment]
I see in Portainer that no images were downloaded for these containers if I am not wrong. Screenshot attached.
![portainer_images](https://github.com/nextcloud/all-in-one/assets/52236863/34d7a1cd-cdf7-4226-b9e8-6f6ec370f838)

**Comment by szaimen** [comment]
Hi, can you post the output of `sudo df -h` here?

**Comment by vawaver** [comment]
![image](https://github.com/nextcloud/all-in-one/assets/52236863/3a2cfb45-b76f-4cb2-a719-001d8687c557)

**Comment by szaimen** [comment]
Hm... can you run `sudo docker image prune -a` then stop the containers and start them again and check if it helps?

**Comment by szaimen** [comment]
BTW, this is your issue: `error creating overlay mount to /var/lib/docker/overlay2/a8f6216d87cac1911f3ffbfe27821bade861410a69312c896b4 (truncated...)`. Last time this happened, the hoster was the problem so moving to discussions. See https://github.com/nextcloud/all-in-one/discussions/1844 for reference.

**Reply by szaimen** [reply]
May I ask which hoster you are using so that we can put it into the readme to disrecommend that one?

**Comment by vawaver** [comment]
I use self-hosted Proxmox, and this is one of six other VMs. Therefore, I did not expect any other problems. I utilize over 15 Docker installations for various apps across my VMs, and so far, there haven't been any issues—until now. As a fan of Docker solutions, I have personally installed the Ubuntu server, and it has been running smoothly for over a year. Other Docker apps also work flawlessly on the server.

The link you sent me was not a solution; it merely provided an alternative type of installation. It does not align with what I am looking for.

**Comment by vawaver** [comment]
> Hm... can you run `sudo docker image prune -a` then stop the containers and start them again and check if it helps?
I restarted the containers from Portainer. It did not help.

**Comment by szaimen** [comment]
Sorry, but I cannot reproduce your problem. How did you install docker btw? Did you use the rwcommended script?

**Comment by vawaver** [comment]
I use this docker-compose.yml
```
version: "3.4"

volumes:
 nextcloud_aio_mastercontainer:
   name: nextcloud_aio_mastercontainer
services:
 nextcloud:
   image: nextcloud/all-in-one:latest
   restart: unless-stopped
   container_name: nextcloud-aio-mastercontainer
   volumes:
     - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
     - /var/run/docker.sock:/var/run/docker.sock:ro
   ports:
     - 8888:8080 # change the port on the left side if it's already in use on your host system.
   environment:
     - APACHE_PORT=11000

**Comment by vawaver** [comment]
> Sorry, but I cannot reproduce your problem. How did you install docker btw? Did you use the recommended script?
I use script installation from this link for all my Ubuntu server installations.
https://gitlab.com/bmcgonag/docker_installs

**Reply by szaimen** [reply]
Can you try to install docker with `curl -fsSL https://get.docker.com | sudo sh` instead? Also what OS are you using?

**Reply by vawaver** [reply, marked as answer]
I would like to offer a sincere apology for the situation. I have identified the source of the problem, which was an error on my part. Specifically, I made a change to the APACHE_PORT variable by assigning it a different value.

However, upon resetting the variable to its original value, the installation proceeded smoothly without any complications whatsoever.

**Reply by szaimen** [reply]
Great that you found the problem!
